README.md

```diff
@@ -34,6 +34,12 @@ To use `latest` images (not recommended) an image tag must be specified explicit
 prebuildify-cross -i linux-armv7:latest -t ..
 ```
 
+Set a custom node_modules path:
+
+```sh
+prebuildify-cross --nodemodulespath ../../node_modules -i linux-armv7:latest -t 20.0.0 --strip
+```
```

Comment:
Would be nice to also document the motivation behind this flag. And `--nodemodulespath` is a bit hard to read. How about just `--modules`?

````suggestion
When working in a monorepo, where `./node_modules` does not contain the dependencies that `prebuildify-cross` relies upon, a custom path can be provided with `--modules`. It can be an absolute path or relative to the current working directory. For example:

```sh
prebuildify-cross --modules ../../node_modules -i linux-armv7:latest -t 20.0.0 --strip
```
````